modules/push-notifications/src/providers/PushOwl.provider.ts

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import { BaseNotificationProvider } from './base.provider.js';
+import { IPushOwlSettings } from '../interfaces/IPushOwlSettings.js';
+import {
+  ISendNotification,
+  ISendNotificationToManyDevices,
+} from '../interfaces/ISendNotification.js';
+import { ConduitGrpcSdk } from '@conduitplatform/grpc-sdk';
+import { NotificationToken } from '../models/index.js';
+
+export class PushOwlProvider extends BaseNotificationProvider<IPushOwlSettings> {
+  private apiKey?: string;
+  private endpoint?: string;
+
+  constructor(settings: IPushOwlSettings) {
+    super();
+    this._initialized = false;
+    this.isBaseProvider = false;
+    this.updateProvider(settings);
+  }
+
+  updateProvider(settings: IPushOwlSettings) {
+    try {
+      this.apiKey = settings.apiKey;
+      this.endpoint = settings.endpoint;
+
+      if (!this.apiKey || !this.endpoint) {
+        throw new Error('Missing required PushOwl API configuration.');
+      }
+
+      this._initialized = true;
+    } catch {
+      this._initialized = false;
+    }
+  }
+
+  async sendMessage(
```

Comment:
notification does not support sending data or silent notifications, is this a provider issue?